runtime/doc/vim_diff.txt

```diff
@@ -28,6 +28,10 @@ these differences.
 ==============================================================================
 2. Option defaults					  *nvim-option-defaults*
 
+- syntax highlighting is enabled by default (can be disabled with the "-u NONE"
+  cmdline argument)
+- filetype-related plugins and scripts are enabled by default (can be disabled
+  with the "-u NONE" cmdline argument)
```

Comment:
perhaps add a newline between these top two and everything below? Also, why not just say `plugins' instead of `plugins and scripts'? They sound the same to me.